tck/src/main/java/org/eclipse/microprofile/lra/tck/TckContextTests.java

```diff
@@ -103,7 +103,7 @@ public void testStatus() throws InterruptedException {
         assertEquals(testName.getMethodName() + ": participant is not completing", ParticipantStatus.Completing.name(), status);
 
         // clear the EndPhase override data so that the next status request returns completed or compensated
-        invoke(false, STATUS_PATH, PUT, lra, 200, ContextTckResource.EndPhase.SUCCESS, 200);
+        invoke(false, STATUS_PATH, HttpMethod.POST, lra, 200, ContextTckResource.EndPhase.SUCCESS, 200);
```

Comment:
sorry if I'm missing something here, but this should call [this method](https://github.com/eclipse/microprofile-lra/pull/172/files#diff-4c219933e42e7b38d324e42b907623a2R342) right? I think it should be GET here and in second clear invocation on line 156.

Reply:
@xstefank you are correct. I separated out the two "status" method to avoid confusion. I also added some other fixes for the "String to URI" change.

Reply:
@xstefank I fixed the PR. I am on PTO tomorrow and need this fix for the narayana implementation so I am going to merge in order to get the snapshot build. I've tested my changes but if you do spot anything please update the issue or PR.